composer.json

```diff
@@ -20,7 +20,7 @@
     ],
     "require": {
         "php": "^8.1",
-        "silverstripe/framework": "^5.3",
+        "silverstripe/framework": "^5.4",
```

Comment:
No need to bump - the config will simply be unused in earlier versions.

Reply:
Updated